src/components/README.md

```diff
@@ -24,3 +24,42 @@ function header = (props) => {
     );
 }
 ```
+# Accordion
+
+### Props
+
+
+| Name |Type | Default | Description  |
+| ------------- |:-------------:| --- | ---- |
+| border|`bool`| false |  Border of the Accordion |
+| children | `Reactnode` |  |Children to render inside Accordion|
+| open|`bool`| false | Determines if the accordion is currently open or closed |
+| title |`string`|  | Title of the Accordion  |
+
+```
+      <div className={`accordion ${this.props.border ? 'border' : ''}`}>
```

Comment:
This for this PR @Abdo-A.
I believe adding the full HTML of the component to the documentation would make the readME too bulky and tough to read.
Good code should be self-documenting so we can skip this part.

We should focus on documenting the props the components use, which is what you have done.
Thanks.

Reply:
I agree @jomadoye ... I think having sample HTML of how to *use* the component could be helpful, but it looks like this an example of the code *generated* by the component?

Are there any best practices for React component comments... I see a generation tool that appears to be popular:

https://github.com/styleguidist/react-styleguidist

For what it's worth, here's the format I came up with for the one component I actually documented:

https://github.com/openmrs/openmrs-react-components/blob/master/src/components/list/List.jsx#L7

I'm not necessarily married to that format (or any other), but would be great to pick a format and standardize on it.